src/app/(proper_react)/redesign/(public)/LandingView.test.tsx

```diff
@@ -306,4 +347,12 @@ describe("When Premium is available", () => {
 
     expect(signIn).toHaveBeenCalledTimes(1);
   });
+
+  it("shows the progress card illustration in the fix your exposures section", () => {
+    const ComposedDashboard = composeStory(LandingUs, Meta);
+    render(<ComposedDashboard />);
+
+    const progressCardIllustration = screen.getByTestId("progress-card-image");
+    expect(progressCardIllustration).toBeInTheDocument();
```

Comment:
Thinking a bit about what bugs might actually occur... This is currently primarily verifying that an image with the right test ID exists in the document. Which is OK, but requires us to do some extra work just for the sake of the test (make sure we use the correct test ID everywhere), and those test IDs don't actually affect the user.

So one thing we could also check for, is whether the Premium-specific content is actually shown (using [`'.toHaveTextContent`](https://github.com/testing-library/jest-dom#tohavetextcontent), and whether non-Premium specific content is _not_ shown (`.not.toHaveTextContent`).

Reply:
Yep, that sounds good :+1: